src/components/ens-registration/RegistrationAvatar/RegistrationAvatar.tsx

```diff
@@ -135,42 +142,38 @@ const RegistrationAvatar = ({
           />
         </Skeleton>
       ) : (
-        <ConditionalWrap
+        <ConditionalContextMenu
           condition={hasSeenExplainSheet}
-          wrap={children => <ContextMenu>{children}</ContextMenu>}
+          onNonConditionPress={onShowExplainSheet}
         >
-          <ButtonPressAnimation
```

Comment:
let me know if this doesn't make sense but before we had a ButtonPressAnimation being rendered all the time even tho we only need it when `!hasSeenExplainSheet` which only happens once

Reply:
Can we not put the `ButtonPressAnimation` inside `wrap`? 

I'm not sure if it belongs in `useSelectImageMenu`